### DISCUSSION TITLE
Authentik as Network Policy Server

### DISCUSSION THREAD

**Discussion by allahshukur-ahmadzada** [goauthentik/authentik#8213] [Q&A]
Hello all, can you please let me clarify what "Currently, only authentication requests are supported." means on https://goauthentik.io/docs/providers/radius/ . Can Authentik be used for network infrastructure authentication? will it return radius attributes so device can do RBAC, or even router devices will be able to authenticate?

**Comment by terafirmanz** [comment]
As I understand it this means no enrolment or other flows can be run only an authentication flow (e.g. user auth).

As for what you say yes device auth but you might want to look at property mappings and user attributes to pass back to the device the attributes you want.
